core-bundle/src/Resources/contao/drivers/DC_Folder.php

```diff
@@ -367,12 +367,7 @@ public function showAll()
 				$for = preg_quote($for);
 			}
 
-			$strPattern = "CAST(name AS CHAR) REGEXP ?";
-
-			if (substr(Config::get('dbCollation'), -3) == '_ci')
-			{
-				$strPattern = "LOWER(CAST(name AS CHAR)) REGEXP LOWER(?)";
-			}
+			$strPattern = "LOWER(CAST(name AS CHAR)) REGEXP LOWER(?)";
```

Comment:
So now we always cast to lower even if the DB collation is case-sensitive?

Reply:
Yes. But this was already the case in 4.13 IMO, because `dbCollation` from the localconfig was not used by Doctrine.